### ISSUE TITLE
[possible bug] armor values of alfsteel dont change ingame after changing them in config

### ISSUE THREAD

**Issue by BennetBfkg** [noeppi-noeppi/MythicBotany#116]
as title says, the newly added config part for alfsteel armor doesnt work, when i change armor or toughness values they dont carry over into game.

**Comment by noeppi-noeppi** [comment]
Did you restart the game after changing the config? And how did you verify the values?

**Comment by BennetBfkg** [comment]
Well, after starting the game, generating the config and changing it I then restarted the game and the newly adjusted values didn't show, only the vanilla ones. I also tried putting it on to see if it was just a visual bug, but the armor bar also only showed the vanilla amount of armor points. I also tried it out with the MMM target dummy mod to make sure it wasn't just a visual bug, and it really seems like the config doesn't impact the in game values at all. To add to that, I was changing some values for the armors in the "gobber 2" mod at the same time, and they worked fine after restarting the game.
